releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,17 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+# spring-cloud-azure-feature-management-web
+
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 1.1.2/1.2.2 - February 24, 2020
+
+* AppConfigurationRefresh's refreshConfigurations() can be called to check if the configurations are not up to date and if the cache has expired trigger a refresh event.
+* A new authentication method has been added. AppConfigurationCredentialProvider and KeyVaultCredentialProvider can be used to pass TokenCredentials to the provider to use when connecting with App Configuration and Key Vault. This method also allows for different authentication methods/credentials to be used when connecting with multiple App Configuration stores or Key Vaults.
+* The fail-fast configuration has been updated to be per store. On startup with fail-fast set to false if an error happens on that store it will be skipped, and will be ignored during refresh. Also now during refresh all stores being refreshed need to be updated successfully. If not successful a new refresh attempt will be made when the cache-expiration date passes.
```

Comment:
When we say "fail-fast configuration", we'd better give full configuration name. This is actually a breaking change, so please list out the before and after.

Customers may not always have the context as we do. For example, what do we mean by "*if an error happens on that store*"? It will be helpful to be specific. We could say something like "if an error happens when loading configuration from a store at application startup".

I am not sure what we are trying to convey in the last sentence. The "cache-expiration *date*" is also confusing.